### ISSUE TITLE
Scroll-to-zoom is backwards

### ISSUE THREAD

**Issue by ianrrees** [WandererFan/FreeCAD-TechDraw#43]
On my machine at least, scrolling up zooms "closer" in the 3D view in FreeCAD, but "further" in the Drawing MDI view.  Should be consistent between the 3D view and the Drawing viewer, which I suppose means we might need to query the 3D viewer's mouse navigation preferences?

**Comment by WandererFan** [comment]
Yes, we are backwards from 3D, but the same as old DrawingView.   Don't
know where this magic happens, but I guess we should match 3D.
